.github/workflows/tox.yml

```diff
@@ -209,6 +209,16 @@ jobs:
             .tox/**/log/
             .tox/**/coverage.xml
 
+      - name: Upload test results to Codecov
+        if: ${{ !cancelled() && hashFiles('junit.xml') != '' }}
+        uses: codecov/test-results-action@v1
+        with:
+          fail_ci_if_error: true
+          name: ${{ matrix.name }}
+          # unable to use wildcards yet due to https://github.com/codecov/test-results-action/issues/110
+          flags: ${{ matrix.python_version }},${{ matrix.os }}
+          use_oidc: true
```

Comment:
Look at my PR on extension that adds it for a better condition for oidc. Also update the merge_group :)

Reply:
Is `merge_group` valid here? This only gets triggered on `workflow_call`, but maybe I'm misunderstanding what you mean.